cli/flox-rust-sdk/src/models/environment/remote_environment.rs

```diff
@@ -26,7 +80,11 @@ impl Environment for RemoteEnvironment {
         packages: Vec<String>,
         flox: &Flox,
     ) -> Result<InstallationAttempt, EnvironmentError2> {
-        todo!()
+        let result = self.inner.install(packages, flox).await?;
+        self.inner
+            .push(false)
+            .map_err(RemoteEnvironmentError::UpdateUpstream)?;
```

Comment:
nonblocking: we could consider deleting the git branch for the temporary environment

Reply:
yeah we create some stale braches, I'd leave that for a separeate issue

Reply:
nonblocking
```suggestion
            .map_err(RemoteEnvironmentError::UpdateUpstream)?;
            // TODO: clean up git branch for temporary environment
```